src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/service/ClientsService.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("DEPRECATION")
```

Comment:
Would remove this warning suppress (especially at the file level). What are you using that is deprecated? And can't we avoid using it?

Reply:
Removed